src/rpc/client/ts/src/server.ts

```diff
@@ -80,22 +51,24 @@ export async function startServer(
     rootPath: rootPath,
     port: port,
   })
-  const artifact = new Artifact('omega-edit-grpc-server', version, rootPath)
 
+  const scriptDir = path.join(rootPath, 'node_modules/omega-edit/bin')
   let args = [`--interface=${host}`, `--port=${port}`]
+
   const logConf = path.resolve('.', 'logconf.xml')
   if (fs.existsSync(logConf)) {
     args.push(`-Dlogback.configurationFile=${logConf}`)
   }
+
   // Start the server
   getLogger().debug(
-    `starting server ${artifact.scriptPath} with args ${args} in directory ${artifact.scriptDir}`
+    `starting server with args ${args} in directory ${scriptDir}`
   )
-  const server = child_process.spawn(artifact.scriptPath, args, {
-    cwd: artifact.scriptDir,
-    stdio: 'ignore',
-    detached: true,
-  })
+
+  const scriptOutput = execSync(
+    `node omega-edit-grpc-server.js ${args.join(' ')}`
```

Comment:
Can we use the same node interpreter that is being used to run _this_ script?  For example `process.argv[0]` instead of using the version of node in the PATH?

Reply:
I am not sure this is possible. Based on documentation they always say to use `node` or put `#!/usr/bin/env node` at the top of the file for linux/unix.